### PR TITLE
[WIP] Fixes Issue 301

### PR DESCRIPTION
**Pull request by lavish205**
some context information is pending.

### CODE REVIEW

**Comment by lavish205** [comment]
I don't know how other two commits are included in this PR

File: wye/reports/views.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
-from django.shortcuts import render
```
Comment (vnbang2003) [inline wye/reports/views.py]: Why this file in pull request  ?

File: wye/templates/reports/index.html [hunk 2]
```diff
@@ -17,6 +17,8 @@ <h3 class="space-1q-top"> Report from {{date}}</h3>
                     <h3 class="space-1q-top"> Number of participants: {{no_of_participants}}</h3>
```
Comment (vnbang2003) [inline wye/templates/reports/index.html]: I think you don't need this files ?

File: wye/templates/workshops/workshop_detail.html [hunk 3]
```diff
@@ -107,7 +107,7 @@ <h1 class="no-space">Workshop Details</h1>
                         <div class="row">
                             <div class="col-xs-2"><strong>College representative</strong></div>
```
Comment (vnbang2003) [inline wye/templates/workshops/workshop_detail.html]: Even this file doesn't need to be in pull request ?

Comment (lavish205) [inline wye/templates/workshops/workshop_detail.html]: I just rebased with master and created PR, IDK how these files are part of this PR

**Pull request closed without merging**